From a4fa3030febbd8ba1efee6ba43735b0b4a5d099f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Cerveau?= <scerveau@igalia.com>
Date: Thu, 23 Mar 2023 15:58:45 +0100
Subject: [PATCH 17/77] pbutils: add video/x-ivf to descriptions

Add missing description for video/x-ivf spotted
with gst-discoverer-1.0

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/4248>
---
 gst-libs/gst/pbutils/descriptions.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gst-libs/gst/pbutils/descriptions.c b/gst-libs/gst/pbutils/descriptions.c
index e5f0275b..ef389d3c 100644
--- a/gst-libs/gst/pbutils/descriptions.c
+++ b/gst-libs/gst/pbutils/descriptions.c
@@ -107,6 +107,7 @@ static const FormatInfo formats[] = {
   {"video/x-quicktime", "Quicktime", AVIS_CONTAINER, "mov"},
   {"video/quicktime", "Quicktime", AVIS_CONTAINER, "mov"},
   {"video/mj2", "Motion JPEG 2000", AVIS_CONTAINER, "mj2"},
+  {"video/x-ivf", "Indeo video format (IVF)", AVIS_CONTAINER, "ivf"},
 
   /* audio formats with static descriptions */
   {"audio/x-ac3", "AC-3 (ATSC A/52)", FLAG_AUDIO, "ac3"},
-- 
2.40.1

